server/src/bin/taskchampion-sync-server.rs

```diff
@@ -27,6 +26,12 @@ async fn main() -> anyhow::Result<()> {
                 .value_parser(ValueParser::os_string())
                 .default_value("/var/lib/taskchampion-sync-server"),
         )
+        .arg(
+            arg!(-C --"allow-client-id" <CLIENT_IDS> "Client IDs to allow (can be repeated; default: all)")
```

Comment:
Shouldn't it be `CLIENT_ID` instead of `CLIENT_IDS`? If I understand correctly, there should be exactly one client ID per flag?